modules/tasks/tasks/src/main/resources/webapp/js/directives.js

```diff
@@ -342,7 +342,7 @@
                         field = ele.data('value');
                         container.append(formatField(field));
                     }
-                }else{
+                } else if (!(this.tagName && this.tagName.toLowerCase() === 'em')) {
```

Comment:
please move this condition into separate method, it will be more readable. This method should be named for example 'isTagForPlaceholder'